web-apps/genesys-spark-examples/scripts/generate-versions-file.mjs

```diff
@@ -25,7 +25,13 @@ const newVersionsJson = semverSort
   .desc([
     ...new Set([].concat(currentVersionsJson, npmVersionsJson, newVersion))
   ])
-  .filter(v => !Boolean(prerelease(v)));
+  .filter(v => {
+    if (v === '5.0.0') {
+      return false;
+    }
+
+    return !Boolean(prerelease(v));
+  });
```

Comment:
Ran command on my local, when comparing the two version files that were outputted, the change appeared to have the desired result.